### ISSUE TITLE
Add existence checks for functions/aliases

### ISSUE THREAD

**Issue by aw1621107** [aw1621107/dotfiles#33]
Specific problem is that when [messing with the brew install](https://github.com/Homebrew/homebrew-core/issues/9484#issuecomment-278080843) it seems that bash is hardcoded to find `fasd` somewhere. This should fail more gracefully.

**Comment by ilovezfs** [comment]
You probably just need to run `hash -r` no?

**Comment by aw1621107** [comment]
No, bash complains even in new sessions. I think it's something to do with the initialization of fasd in the dotfiles, since it uses `/usr/local/bin/fasd --init` or something of the sort, if I remember correctly.
